.github/workflows/update_submodules.yml

```diff
@@ -0,0 +1,29 @@
+name: Update Submodules
+
+on:
+  repository_dispatch:
+    types: [update_submodules]
+
+jobs:
+  update:
+    permissions:
+      contents: write
+      actions: write
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Update submodules
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git submodule update --init && git submodule update --remote
```

Comment:
on bitcointranscripts, we are also [using](https://github.com/bitcointranscripts/bitcointranscripts/blob/master/.github/workflows/sync-parent-repo-with-submodule-updates.yml#L28-L29) the `--recursive` flag

is this something that we might need?

Reply:
Does the `recursive` on line 17 cover this ?

Reply:
Yes, the checkout with recursive covers this.

It isn't necessary either as recursive deals with nested submodules. And none of the submodules have nested submodules. But it doesn't hurt to have it there just in case